tests/ContainsManyTest.php

```diff
@@ -266,6 +267,10 @@ public function testNestedContainsMany()
         // do we also correctly calculate discounts from nested containsMany?
         $this->assertSame(24.2 * 15 / 100 + 86.25 * 20 / 100, $i->get('discounts_total_sum')); // =20.88
 
+        if ($this->getDatabasePlatform() instanceof OraclePlatform) {
+            $this->markTestIncomplete('TODO - fix CLOB support on Oracle');
```

Comment:
isn't this fixed in other place where you had get_stream call?

Reply:
CLOB - not working, I did fixed stream to string transform in dsql, but still need more work